### ISSUE TITLE
Can I just train the intra type without the cross type？

### ISSUE THREAD

**Issue by wuxiaolianggit** [DeepMotionEditing/deep-motion-editing#65]
excuse me,Can I just train the intra type without the cross type? @kfiraberman

**Comment by PeizhuoLi** [comment]
Of course. But you'll need to modify the code. I'd suggest to start with architecture.py

**Comment by wuxiaolianggit** [comment]
The number of bone points is the same, but the proportion is not the same. I can directly copy the displacement and rotation vector of character a to the corresponding node of character B. in this way, the same effect can be achieved without using deep learning. Do you think my understanding is correct? @PeizhuoLi
